view/adminhtml/templates/boltpay/js/replacejs.phtml

```diff
@@ -475,42 +421,6 @@ require([
     });
     ////////////////////////////////////////////////
 
-    //////////////////////////////////////////////////////////
-    // loop through selectors array and set up the replacement
-    //////////////////////////////////////////////////////////
-    for (var i = 0, length = settings.selectors.length; i < length; i++) {
-        var selector = settings.selectors[i];
-        ! function(selector) {
-            var parts = selector.split('|'),
-                // the CSS selector
-                identifier = parts[0].trim(),
-                // button placement regarding the selector element, prepend, append
-                position =  parts[1];
-            /////////////////////////////////////////////////////
-            // replace the selectors with bolt button identifiers
-            // if / when selectors are in the DOM
-            /////////////////////////////////////////////////////
-            onElementReady(identifier, function(element) {
-                if (getCheckoutKey() === '') return;
-                var bolt_button = document.createElement("div");
-
-                bolt_button.setAttribute("class", bolt_button_css_class);
-                // place the button before or after selector element
-                if (position && position.trim().toLowerCase() === 'append') {
-                    element.parentNode.insertBefore(bolt_button, element.nextSibling);
-                } else {
-                    element.parentNode.insertBefore(bolt_button, element);
-                }
-                // if no position is specified remove the selector element
-                if (!position) {
-                    element.parentNode.removeChild(element);
-                }
-            });
-            /////////////////////////////////////////////////////
-        }(selector);
-    }
-    //////////////////////////////////////////////////////////
-
     ///////////////////////////////////////////////////////////
```

Comment:
We need this. It's how minicart button works. Moreover, it gives us flexibility of customization in checkout button placement.

Reply:
@danac-gs this code is for adminhtml. I donot think admin has minicart?